idaes/models/properties/general_helmholtz/helmholtz_state.py

```diff
@@ -570,6 +570,10 @@ def mole_frac_comp_rule(b, i):
         self.dens_mol_crit = pyo.Expression(
             expr=params.dens_mass_crit / params.mw, doc="critical mole density"
         )
+        self.dens_mol_star = pyo.Expression(
```

Comment:
This looks OK, but I think you need to add `dens_mol_star` to the metadata as well.